docs/src/main/asciidoc/validation.adoc

```diff
@@ -128,7 +128,7 @@ Let's now create the `Result` class as an inner class:
 
 [source, java]
 ----
-private class Result {
+public class Result {
```

Comment:
We should also make it `static`. The quickstart is OK btw, I just checked it.

Reply:
Oh, is it so? Making it public but not static just worked fine for me...am I missing something?

Reply:
It works but it's not optimal.